kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -22,7 +22,7 @@ Export-Package: org.eclipse.kura;version="1.2.0",
  org.eclipse.kura.data.transport.listener;version="1.0.0",
  org.eclipse.kura.db;version="1.0.0",
  org.eclipse.kura.linux.udev;version="1.0.0",
- org.eclipse.kura.message;version="1.0.0",
+ org.eclipse.kura.message;version="1.0.1",
```

Comment:
@dwoodard1 The change in api determined an overloading of KuraTopic method. In my opinion, this implies that the api version for the message package should be increased to 1.1.0.